azurerm/internal/services/postgres/postgresql_flexible_server_resource.go

```diff
@@ -337,25 +333,28 @@ func resourcePostgresqlFlexibleServerRead(d *pluginsdk.ResourceData, meta interf
 	if props := resp.ServerProperties; props != nil {
 		d.Set("administrator_login", props.AdministratorLogin)
 		d.Set("zone", props.AvailabilityZone)
-		if props.SourceServerName != nil && props.SourceSubscriptionID != nil && props.SourceResourceGroupName != nil {
-			d.Set("source_server_id", parse.NewFlexibleServerID(*props.SourceSubscriptionID, *props.SourceResourceGroupName, *props.SourceServerName).ID())
-		}
 		d.Set("version", props.Version)
-		d.Set("cmk_enabled", props.ByokEnforcement)
 		d.Set("fqdn", props.FullyQualifiedDomainName)
-		d.Set("public_network_access_enabled", props.PublicNetworkAccess == postgresqlflexibleservers.ServerPublicNetworkAccessStateEnabled)
+		d.Set("public_network_access_enabled", props.Network.PublicNetworkAccess == postgresqlflexibleservers.ServerPublicNetworkAccessStateEnabled)
 
-		if props.DelegatedSubnetArguments != nil {
-			d.Set("delegated_subnet_id", props.DelegatedSubnetArguments.SubnetArmResourceID)
+		if props.SourceServerResourceID != nil {
+			d.Set("source_server_id", props.SourceServerResourceID)
+		}
```

Comment:
Here, I have tested the "PITR" test case, it shows that in GA version, it returns "source_server_id" empty string. I would remove these lines to avoid the plan diff is not empty.

Reply:
Good to know, removed it. Thanks.